src/frontend/src/handler/alter_sink_config.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright 2025 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::collections::BTreeMap;
+
+use pgwire::pg_response::{PgResponse, StatementType};
+use risingwave_common::bail;
+use risingwave_sqlparser::ast::ObjectName;
+
+use super::{HandlerArgs, RwPgResponse};
+use crate::Binder;
+use crate::catalog::root_catalog::SchemaPath;
+use crate::error::Result;
+
+pub async fn handle_alter_sink_config(
+    handler_args: HandlerArgs,
+    table_name: ObjectName,
+    config: BTreeMap<String, String>,
+) -> Result<RwPgResponse> {
```

Comment:
please add check for:

If the sink is created using CONNECTION, then the in-coming change cannot modify the props defined in the CONNECTION.